### ISSUE TITLE
Feature request - color all lines in rainbow

### ISSUE THREAD

**Issue by bhbmaster** [k-five/bline#1]
A very smart developer "dteske" showed me their tool once that did this. I was never able to find anything close to that. However, bline + lolcat come close. Can you add a feature to bline that colors each line of text in rainbow, always starting with one color and proceeding thru the colors the same way.

Why? For me, it was EXTERMELY useful in viewing logs or text with long lines. My current alternative is to use `command-that-has-long-output | nl`. The nl numbers the lines, and each new line is tabbed over and has a line number, so its easy to see where it starts. With rainbow view it was cooler to see.

Below we can see lolcat (not dteske's tool), the only issue is that each line starts with a different color, so its hard to tell when a new line starts

<img width="370" alt="image" src="https://user-images.githubusercontent.com/62363157/185725421-e831be10-cc94-4949-b116-297dc138382d.png">

**Comment by shakibamoshiri** [comment]
@bhbmaster if I understood your idea correctly you would like to see each long line separately from others long lines? so reading long line logs be easier for eyes? (please correct me if I am wrong)

For the **feature request** technically it is possible (should be) but currently my focus in not on coding in `C` and not to mention this is for - I guess - more then 4 years ago, so takes time to review `C` and my code

### lolcat
It is a tool to deal with a single letter while `bline` deals with lines. Yes I did not think about it, but it could be a rainbow line. Anyway you can colorize just **odd** or **even** lines to have a more readable output
```bash
man git | bline -o yellow
```

### rainbow
I am pretty sure your idea (if I understood it correctly) is possible to be done using `bash` and `lolcat` that you parse the input line by line and each line is send to `lolcat` so you would have rainbow line. Just you should make sure disabling `lolcat` random color

**Comment by bhbmaster** [comment]
I totally understand and I am sorry. I actually just tried lolcat again and it does the trick. I just misremembered it I guess.
I was concerned that when new line starts it starts with a slightly new color which would be hard to keep track of what I was wanting. However, that does not take away from being able to tell when a line is wrapped to a new line because its too long, versus it just being an actual new line `\r` or `\n`. As we can see in the screenshot below.

![image](https://user-images.githubusercontent.com/62363157/185885455-77833459-600a-4d98-9e97-23db072158d5.png)

I submitted this feature request to lolcat and to your project because I was not sure who would get back to me first.

I just closed both cases (the one here and there)
